auth/lib/models/code.dart

```diff
@@ -81,12 +86,26 @@ class Code {
   }
 
   static Code fromRawData(String rawData) {
+    bool isSteam = false;
+    if (rawData.startsWith("otpauth://steam")) {
+      rawData = rawData.replaceAll("://steam", "://totp");
```

Comment:
changing the issuer here will break export. Ideally, on export we should be exporting it as otpauth://stream

Reply:
Done!